cc3d/player5/Plugins/ViewManagerPlugins/ScreenshotManager.py

```diff
@@ -259,16 +237,29 @@ def ok_to_add_screenshot_detailed(self, scr_data: ScreenshotData) -> bool:
                 ret = QMessageBox.warning(
                     self.tabViewWidget(),
                     "Screenshot Already Exists",
-                    "Screenshot with identical settings exists in <b>screenshot_data/screenshots.json</b><br/>"
+                    "Screenshot with identical settings exists in <br/><b>screenshot_data/screenshots.json</b><br/>"
                     "Would you like to add with different screenshot name or discard it (recommended) ",
                     QMessageBox.No | QMessageBox.Yes,
                 )
 
                 if ret == QMessageBox.No:
-                    return False
+                    ok_to_add_screenshot = False
 
-            print
-        return True
+
+        if ok_to_add_screenshot:
```

Comment:
This is an improvement. Previously, we had both a method and variable named the same: `ok_to_add_screenshot`.